packages/runner-docker/scripts/build.mjs

```diff
@@ -0,0 +1,129 @@
+#!/usr/bin/env node
```

Comment:
Honestly it was files like this that I expected to see, would you be able to extract *just* the changes that make the local file stuff use JS instead of bash into a new PR? That would really help make this stuff easier to review and land.